components/core/tools/packaging/common/bundle-libs.sh

```
@@ -128,7 +128,7 @@ for bin in "${BINARIES[@]}"; do
     [[ -f "${bin_path}" ]] || continue
 
     cp "${bin_path}" "${DESTDIR}${prefix}/bin/${bin}"
-    patchelf --set-rpath "${lib_install_dir}" "${DESTDIR}${prefix}/bin/${bin}"
+    patchelf --set-rpath '$ORIGIN/../lib/clp' "${DESTDIR}${prefix}/bin/${bin}"
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)